src/features/modals/CreateProjectModal.tsx

```diff
@@ -1,59 +1,57 @@
-import { signal } from "@preact/signals-core";
-import { type Ref, useRef, useState } from "preact/hooks";
+import { useRef, useState } from "preact/hooks";
 import { PlusIcon } from "../icons/PlusIcon";
 import { create, useProjectsList } from "../../lib/projects";
+import { Modal } from "./Modal";
 
-const refSignal = signal<Ref<HTMLDialogElement> | null>(null);
-
-export const CreateProjectButton = () => (
-  <div className="tooltip" data-tip="Create new project">
-    <button
-      class="btn btn-sm btn-neutral btn-circle"
-      onClick={() => refSignal.value?.current?.showModal()}
-    >
-      <PlusIcon />
-    </button>
-  </div>
-);
-
-export const CreateProjectModal = () => {
+export const CreateProjectModalButton = () => {
   const { refetch } = useProjectsList();
-  refSignal.value = useRef<HTMLDialogElement>(null);
+  const dialog = useRef<HTMLDialogElement>(null);
```

Comment:
Daisy's modal uses a native dialog, which works with DOM level methods to open and close.